Add intros for articles on commits (#7328)
github/docs: content/github/committing-changes-to-your-project/viewing-and-comparing-commits/differences-between-commit-views.md

---
title: Differences between commit views

> intro: You can sometimes observe differences in commit history depending on the chosen viewing method.

Reviewer: ```suggestion
intro: You might observe differences in commit history depending on the chosen viewing method.
```
